commit 89f64aed8ebf0d7aaef0323f3fba2b16a29cbbad
Author: onettboots <blackcocopet@gmail.com>
Date:   Wed Apr 26 11:31:29 2023 +0000

    [REAPPLY] kernelsu: we're non GKI and non KPROBES build
    
    Change-Id: Ib2fd3e1c5beb3f0e98772c2458e3a6c9485920dc
    Signed-off-by: Angga <angga@linuxmail.org>
    Signed-off-by: Mayur <ultramayur123@gmail.com>

diff --git a/drivers/kernelsu/ksu.c b/drivers/kernelsu/ksu.c
index c2da89022624..1dc3901ba932 100644
--- a/drivers/kernelsu/ksu.c
+++ b/drivers/kernelsu/ksu.c
@@ -53,12 +53,12 @@ int __init kernelsu_init(void)
 
 	ksu_uid_observer_init();
 
-#ifdef CONFIG_KPROBES
+//#ifdef CONFIG_KPROBES
 	ksu_enable_sucompat();
 	ksu_enable_ksud();
-#else
-	pr_alert("KPROBES is disabled, KernelSU may not work, please check https://kernelsu.org/guide/how-to-integrate-for-non-gki.html");
-#endif
+//#else
+//	pr_alert("KPROBES is disabled, KernelSU may not work, please check https://kernelsu.org/guide/how-to-integrate-for-non-gki.html");
+//#endif
 
 	return 0;
 }
diff --git a/drivers/kernelsu/ksu.h b/drivers/kernelsu/ksu.h
index dae32738a899..f2cd14effba2 100644
--- a/drivers/kernelsu/ksu.h
+++ b/drivers/kernelsu/ksu.h
@@ -5,13 +5,13 @@
 #include "linux/workqueue.h"
 
 #ifndef KSU_GIT_VERSION
-#warning                                                                       \
-	"KSU_GIT_VERSION not defined! It is better to make KernelSU a git submodule!"
-#define KERNEL_SU_VERSION (16)
-#else
-#define KERNEL_SU_VERSION                                                      \
-	(10000 + KSU_GIT_VERSION +                                             \
-	 200) // major * 10000 + git version + 200 for historical reasons
+//#warning                                                                       \
+//	"KSU_GIT_VERSION not defined! It is better to make KernelSU a git submodule!"
+#define KERNEL_SU_VERSION (10818)
+//#else
+//#define KERNEL_SU_VERSION                                                      \
+//	(10000 + KSU_GIT_VERSION +                                             \
+//	 200) // major * 10000 + git version + 200 for historical reasons
 #endif
 
 #define KERNEL_SU_OPTION 0xDEADBEEF
